godtools/Managers/TranslationFileRemover.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  TranslationFileRemover.swift
+//  godtools
+//
+//  Created by Ryan Carlson on 5/19/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import Crashlytics
+
+class TranslationFileRemover: GTDataManager {
+    
+    func deleteUnusedPages() {
+        for page in findAllEntities(ReferencedFile.self) {
+            do {
+                if page.translationsAsArray().filter({ $0.isDownloaded }).count == 0 {
+                    try FileManager.default.removeItem(atPath: "\(resourcesPath)/\(page.filename!)")
+                }
+            } catch {
+                Crashlytics().recordError(error, withAdditionalUserInfo: ["customMessage": "Error deleting file \(page.filename!) no longer referenced by any translations"])
```

Comment:
How about we consider this as a task for later? I definitely want to make sure that we're doing proper error handling throughout the project. We didn't in MH and that's made debugging painful. However there's other more pressing that need working out before we come back to this in my opinion.